frontend/src/index.tsx

```diff
@@ -12,6 +12,8 @@ import {
 import { AppRoute } from '~/bundles/common/enums/enums.js';
 import { store } from '~/framework/store/store.js';
 
+import { NotFound } from './bundles/common/pages/pages.js';
```

Comment:
Small detail, but better to change to match the code style of the project
```suggestion
import { NotFound } from '~/bundles/common/pages/pages.js';
```